xde-tests/src/lib.rs

```diff
@@ -54,19 +115,95 @@ impl OpteZone {
     /// of interfaces. In illumos parlance, the interfaces are data link
     /// devices.
     fn new(name: &str, zfs: &Zfs, ifx: &[&str], brand: &str) -> Result<Self> {
+        // Ensure any prior zone with this name is fully removed before creating
+        // a new one, to avoid flakes from leftover state.
+        let _ = ensure_zone_absent(name);
         let zone = Zone::new(name, brand, zfs, ifx, &[])?;
         Ok(Self { zone })
     }
 
-    /// Wait for the network to come up, then set up the overlay network.
+    /// Wait for the network to come up, then set up the IPv4 overlay network.
     fn setup(&self, devname: &str, addr: String) -> Result<()> {
         self.zone.wait_for_network()?;
+        // Configure IPv4 with static address (immediate, no DHCP wait)
+        self.zone.zexec(&format!(
+            "ipadm create-addr -t -T static -a {addr}/24 {devname}/test"
+        ))?;
```

Comment:
I'm not sure we want to make this static -- it may be faster, but we're losing out on a) regularly testing that DHCP works against an OS network stack in our own CI and b) making sure that hairpin packet processing proceeds without issue*. Was the delay here causing bad interactions with one or more of the tests?

* ARP fills the same niche for b), I suppose.

Reply:
Yeah. I moved toward static for testing initially, but could move this back to DHCP.